contracts/src/Types.sol

```diff
@@ -84,11 +84,13 @@ enum Command {
     SetOperatingMode,
     TransferNativeFromAgent,
     SetTokenTransferFees,
-    SetPricingParameters
+    SetPricingParameters,
+    RegisterForeignToken
 }
 
 enum AgentExecuteCommand {
```

Comment:
I'm not sure its worth having an `AgentExecuteCommand` enum anymore. `TransferToken` and `MintToken` could just be normal commands.

Reply:
Yeah, I think so and will refactor accordingly.

Reply:
https://github.com/Snowfork/snowbridge/pull/1155/commits/b2a666ac836fd926e087580ace88acb98e58a0d4